Rakefile

```diff
@@ -29,5 +29,5 @@ end
 
 require 'yard'
 YARD::Rake::YardocTask.new do |t|
-  t.files = ['lib/**/*.rb', 'README.md']
+  t.files = ['lib/**/*.rb', '-', 'README.md']
```

Comment:
The Ruby source files (and any extra documentation files separated by '-') to process.
https://www.rubydoc.info/gems/yard/0.8.6.1/YARD/Rake/YardocTask